joern-cli/build.sbt

```diff
@@ -131,4 +131,26 @@ generateScaladocs := {
 
 Universal / packageBin / mappings ++= sbt.Path.directory(new File("joern-cli/src/main/resources/scripts"))
 
+lazy val removeModuleInfoFromJars = taskKey[Unit]("remove module-info.class from dependency jars - a hacky workaround for a scala3 compiler bug https://github.com/scala/scala3/issues/20421")
+removeModuleInfoFromJars := {
+  import java.nio.file.{Files, FileSystems}
+  val logger = streams.value.log
+  val libDir = (Universal/stagingDirectory).value / "lib"
+
+  // remove all `/module-info.class` from all jars
+  Files.walk(libDir.toPath)
+    .filter(_.toString.endsWith(".jar"))
+    .forEach { jar =>
+      val zipFs = FileSystems.newFileSystem(jar)
+      zipFs.getRootDirectories.forEach { zipRootDir =>
+        Files.list(zipRootDir).filter(_.toString == "/module-info.class").forEach { moduleInfoClass =>
```

Comment:
does this comparison with a `/` work on Windows?

Reply:
![grafik](https://github.com/user-attachments/assets/e2985180-d975-4af2-b9f9-2776483088fb)
just updating crowdstrike, won't be long :)

Reply:
yes it works on windows